src/controllers/guests/posts.rs

```diff
@@ -167,14 +123,10 @@ pub async fn get_category_posts(
             .content_type(ContentType::html())
             .finish());
     } else {
```

Comment:
Please check up coding convention to refactor this part
```rust
// Bad practice
if ... {
    [...]
    return ...
}else {
   [...]
}
```

Reply:
fixed it